python/ray/autoscaler/node_provider.py

```diff
@@ -70,6 +70,9 @@ def import_external():
     """Mock a normal provider importer."""
 
     def return_it_back(config):
+        if config["provider"]["custom_bootstrap_config"]:
+            provider_cls = load_class(path=config["provider"]["module"])
+            config = provider_cls.bootstrap_config(config)
```

Comment:
Why not always load the class, and then do `if hasattr(provide_cls, "bootstrap_config")`.

Actually can we make bootstrap_config() part of the node provider interface and always call it? Not sure why we didn't do it this way in the beginning for all node providers actually.

Reply:
Sounds good!